app/src/main/java/com/kamron/pogoiv/MainActivity.java

```diff
@@ -340,11 +340,14 @@ public void onStop() {
     protected void onResume() {
         super.onResume();
 
-        //if (BuildConfig.enableUpdater) {
-            //if (currentSettings.isUpdatesEnabled()) {
-                new AppUpdateLoader().start();
-                checkingForUpdate = true;
-    //}
+        if(BuildConfig.FLAVOR.equals("offline"))
+        {
+            Toast.makeText(mContext, "Offline version does not auto update", Toast.LENGTH_SHORT).show();
```

Comment:
IIUC, this is a message for the user of the offline version every time he resumes the GoIV activity. It's a floating message designed to be unobtrusive, but that still seems too much noise to me. Or am I missing sth.?